PhotoKingdomAPI/PhotoKingdomAPI/Controllers/Manager.cs

```diff
@@ -537,26 +537,26 @@ public int loadData()
 
 
                     // AttractionPhotowars - new photowars
-                    var photowar1 = ds.AttractionPhotowars.Add(new AttractionPhotowar
-                    {
-                        AttractionId = cntower.Id
-                    });
+                    //var photowar1 = ds.AttractionPhotowars.Add(new AttractionPhotowar
+                    //{
+                    //    AttractionId = cntower.Id
+                    //});

```

Comment:
Remove current Photowar seed data for CN Tower
Add Photowar Queue seed data for CN Tower
> Current Photowar will be added from the Queue by "api/CheckOwns"